modules/apps/bookmarks/bookmarks-service/service.xml

```diff
@@ -25,14 +25,14 @@
 
```

Comment:
I also removed `uad-anonymize-field-name="screenName"` on userName as that is not needed. ServiceBuilder generate that part automatically